### PR TITLE
[CDF-7803] Add command to start jobs

### PR DESCRIPTION
**Pull request by sebbert**

### CODE REVIEW

File: Cognite.Jetfire.Cli/Transform/TransformCommand.cs [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+                var cancellationToken = cancellationTokenSource.Token;
+                try
+                {
+                    cancellationTokenSource.CancelAfter(timeout);
+
+                    while (!await IsJobCompleted(client, configId, jobUuid, cancellationToken))
+                    {
+                        await Task.Delay(interval, cancellationToken);
+                    }
+                    Console.Out.WriteLine("Job completed successfully");
```
Comment (wjoel) [inline Cognite.Jetfire.Cli/Transform/TransformCommand.cs:129]: This should be using `console` like the others, I think? Or is it intentionally using `Console` instead for the successful case?

File: README.md [hunk 2]
```diff
@@ -119,6 +119,30 @@ apiKey:
   write: WRITE_API_KEY
 ```
 
+### Start a transformation job
+
+`jetfire-cli` also provides a `transform` subcommand, which can start transformation jobs and/or wait for jobs to complete.
```
Comment (wjoel) [inline README.md:124]: Maybe `run` would have been better but not too important.

**Review by wjoel — APPROVED** [review]
Nice!

File: Cognite.Jetfire.Cli/Transform/TransformCommand.cs (hunk 1 above)
Comment (sebbert) [reply Cognite.Jetfire.Cli/Transform/TransformCommand.cs:129]: Ah, good catch! That's def. an oversight.

File: README.md (hunk 2 above)
Comment (sebbert) [reply README.md:124]: I realize now that I actually used the word `run` when sketching out the interface. I like that better than `transform` 👍

**Pull request merged**